cves/CVE-2017-11719.yml

```
@@ -308,17 +348,19 @@ lessons:
     applies:
     note:
   distrust_input:
-    applies:
-    note:
+    applies: true
+    note: The functions were trusting the mb_height value read in.
   security_by_obscurity:
     applies:
     note:
   serial_killer:
     applies:
     note:
   environment_variables:
-    applies:
-    note:
+    applies: true
+    note: |
+      Based on the if statement the data_offset value if either set or not to a
+      specified address.
```

This might not be a correct fix, but as is I'm a bit confused by what this is trying to get across
```suggestion
      Based on the 'if' statement the data_offset value is either set or not to a
      specified address.
```